### ISSUE TITLE
Does Monzo support domain name as Regex?

### ISSUE THREAD

**Issue by srinicrick65** [monzo/egress-operator#23]
Does Monzo support domain name as Regex? I am to set the domain dns name as ^global\\.azure-devices-provisioning\\.net$

**Comment by jackkleeman** [comment]
No, sorry. Each envoy gateway deployment only supports a specific domain name, as otherwise we'd have to rely on inspection to find the destination
